api/receiver.py

```diff
@@ -563,6 +557,8 @@ def on_request_thread_safe(self, properties, body_raw: bytes):
                     f.write(new_line)
                 else:
                     f.write(line)
+        if config_reloaded:
+            raise ConfigReloadedException
```

Comment:
Why an exception if we (successfully?) reloaded config?

Reply:
When I was testing, simply calling `stop_consuming()` from within `on_request_thread_safe()` just made the receiver unresponsive without restarting the connection, so I used an exception to break out of `start_consuming()`. But this must have been some error on my end, it seems to work now.